segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/RecoveryProcessor.java

```diff
@@ -203,7 +217,7 @@ protected void recoverOperation(DataFrameRecord<Operation> dataFrameRecord, Oper
         }
     }
 
-    private void recordTruncationMarker(DataFrameRecord<Operation> dataFrameRecord) {
+    protected void recordTruncationMarker(DataFrameRecord<Operation> dataFrameRecord) {
```

Comment:
Why is this change needed?

Reply:
leftover from previous change which is removed now. wil revert this.